src/app/zap-templates/zcl/data-model/chip/global-enums.xml

```diff
@@ -15,24 +15,29 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/energy_management.adoc src/app_clusters/meas_and_sense.adoc src/data_model/Data-Model.adoc src/service_device_management/DistributedComplianceLedger.adoc
+Parameters: in-progress 
+Git: 0.7-summer-2025
+-->
+<!--
 TODO: Make these structures global rather than defining them for each cluster.
  This depends on the ability to define global structs via XML tags.
  see: https://github.com/project-chip/connectedhomeip/issues/29818
 -->
-<configurator>
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
   <domain name="CHIP"/>
-
   <enum name="AtomicRequestTypeEnum" type="enum8">
     <item name="BeginWrite" value="0x00"/>
     <item name="CommitWrite" value="0x01"/>
     <item name="RollbackWrite" value="0x02"/>
   </enum>
 
   <enum name="ThreeLevelAutoEnum" type="enum8">
-    <item name="Low" value="0x00"/>
-    <item name="Medium" value="0x01"/>
-    <item name="High" value="0x02"/>
-    <item name="Automatic" value="0x03"/>
+    <item name="Auto" value="0x00"/>
```

Comment:
Is this provisional? Globals make it hard to figure out where things originale from. Maybe we can ask alchemy to somehow hightligh section/usage for easier review

Reply:
Camera is the first cluster to use `ThreeLevelAutoEnum` - it's a global to be used by any cluster, but currently only used by cameras (& introduced with cameras)

Reply:
For reference: https://github.com/search?q=repo%3Aproject-chip%2Fconnectedhomeip+ThreeLevelAutoEnum+language%3AXML&type=code&l=XML